src/ad-hoc-visualizations/accessible-names/visualization.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { AdHocTestkeys } from 'common/configs/adhoc-test-keys';
+import { TestMode } from 'common/configs/test-mode';
+import { VisualizationConfiguration } from 'common/configs/visualization-configuration';
+import { Messages } from 'common/messages';
+import { TelemetryDataFactory } from 'common/telemetry-data-factory';
+import { VisualizationType } from 'common/types/visualization-type';
+import { generateUID } from 'common/uid-generator';
+import { adhoc as content } from 'content/adhoc';
+import { RuleAnalyzerConfiguration } from 'injected/analyzers/analyzer';
+import { ScannerUtils } from 'injected/scanner-utils';
+import { VisualizationInstanceProcessor } from 'injected/visualization-instance-processor';
+import { isEmpty } from 'lodash';
+
+const { guidance } = content.headings;
+const accessiblenamesTestKey = AdHocTestkeys.AccessibleNames;
+
+const headingsRuleAnalyzerConfiguration: RuleAnalyzerConfiguration = {
```

Comment:
nit: it seems like you're leaving this here because you need _some_ configuration to be passed in, which is fine for now, but I would look to rename the variable appropriately and additionally replace the testType below with the appropriate one for this test.